packages/core/_src/stm/TQueue/operations/shutdown.ts

```diff
@@ -0,0 +1,11 @@
+import { concreteTQueue } from "@effect/core/stm/TQueue/operations/_internal/InternalTQueue"
+
+/**
+ * Shuts down the queue.
+ *
+ * @tsplus getter ets/TQueue shutdown
+ */
+export function shutdown<A>(self: TQueue<A>): USTM<void> {
+  concreteTQueue(self)
+  return STM.Effect((journal) => self.ref.unsafeSet(null, journal))
```

Comment:
Perhaps it's better to use `undefined` here so that we can use the `==` trick I mentioned previously.